### ISSUE TITLE
I want to add a new project of mine, how do I do so?

### ISSUE THREAD

**Issue by aarushiksk** [tuvtran/project-based-learning#456]
### Description: My project has not been added before in this repo. I am looking forward to adding it, how should I do so? should I just fork it and add my project (I am a beginner here)

**Comment by Pawnpusher1245** [comment]
Certainly! Here are step-by-step instructions for adding your changes to a project on GitHub, following the provided contribution guidelines:

**Step 1: Fork the Repository**

1. Click the "Fork" button in the upper right corner of the repository's page.
2. This will create a copy of the repository in your own GitHub account.

**Step 2: Clone Your Fork**

1. Open your terminal or command prompt.
2. Clone your forked repository to your local machine using Git. Replace `<your-username>` with your GitHub username and `<repository-name>` with the repository's name:
   ```
   git clone https://github.com/<your-username>/<repository-name>.git
   ```

**Step 3: Create a New Branch**

1. Navigate to the cloned repository:
   ```
   cd <repository-name>
   ```
2. Create a new branch for each tutorial or change you want to make. Use a descriptive branch name:
   ```
   git checkout -b your-branch-name
   ```

**Step 4: Make Your Changes**

1. Ensure that the tutorial(s) you want to add do not already exist in the repository.
2. Place your tutorial(s) correctly under the appropriate language/technology folder.
3. If you are making improvements to an existing tutorial, ensure it is in the correct location and marked as an improvement.

**Step 5: Commit Your Changes**

1. Add your changes to the staging area:
   ```
   git add .
   ```
2. Commit your changes with a clear and descriptive commit message:
   ```
   git commit -m "Add [Title] tutorial"
   ```

**Step 6: Push Your Changes**

1. Push your local branch to your fork on GitHub:
   ```
   git push origin your-branch-name
   ```

**Step 7: Create a Pull Request (PR)**

1. Go to your forked repository on GitHub.
2. Click the "New Pull Request" button for the branch you just pushed.
3. Ensure that the PR title is descriptive and follows the guidelines.

**Step 8: Review and Discuss**

1. Engage in any discussions or feedback provided by the maintainers of the repository.
2. Make any necessary modifications to your PR based on the feedback.

**Step 9: Merge Your Pull Request**

1. Once your changes are approved and meet the guidelines, the maintainers will merge your PR into the main branch.

**Comment by Pawnpusher1245** [comment]
Step 1: Fork the Repository

    Click the "Fork" button in the upper right corner of the repository's page.
    This will create a copy of the repository in your own GitHub account.

Step 2: Clone Your Fork

    Open your terminal or command prompt.
    Clone your forked repository to your local machine using Git. Replace <your-username> with your GitHub username and <repository-name> with the repository's name:

    git clone https://github.com/<your-username>/<repository-name>.git

Step 3: Create a New Branch

    Navigate to the cloned repository:

    cd <repository-name>

    Create a new branch for each tutorial or change you want to make. Use a descriptive branch name:

    git checkout -b your-branch-name

Step 4: Make Your Changes

    Ensure that the tutorial(s) you want to add do not already exist in the repository.
    Place your tutorial(s) correctly under the appropriate language/technology folder.
    If you are making improvements to an existing tutorial, ensure it is in the correct location and marked as an improvement.

Step 5: Commit Your Changes

    Add your changes to the staging area:

    git add .

    Commit your changes with a clear and descriptive commit message:

    git commit -m "Add [Title] tutorial"

Step 6: Push Your Changes

    Push your local branch to your fork on GitHub:

    git push origin your-branch-name

Step 7: Create a Pull Request (PR)

    Go to your forked repository on GitHub.
    Click the "New Pull Request" button for the branch you just pushed.
    Ensure that the PR title is descriptive and follows the guidelines.

Step 8: Review and Discuss

    Engage in any discussions or feedback provided by the maintainers of the repository.
    Make any necessary modifications to your PR based on the feedback.

Step 9: Merge Your Pull Request

    Once your changes are approved and meet the guidelines, the maintainers will merge your PR into the main branch.

**Comment by aarushiksk** [comment]
Thanks a bunch! Really appreciate it :)
